package/MDAnalysis/analysis/align.py

```diff
@@ -359,27 +406,152 @@ def alignto(mobile, reference, select="all", mass_weighted=False,
     ref_coordinates = ref_atoms.positions - ref_com
     mobile_coordinates = mobile_atoms.positions - mobile_com
 
-    old_rmsd = rms.rmsd(mobile_coordinates, ref_coordinates)
-
-    R, new_rmsd = rotation_matrix(mobile_coordinates, ref_coordinates, weights=weights)
+    old_rmsd = rms.rmsd(mobile_coordinates, ref_coordinates, weights)
 
     if subselection is None:
-        atoms = mobile.universe.atoms
+        mobile_atoms = mobile.universe.atoms
     elif type(subselection) is str:
-        atoms = mobile.select_atoms(subselection)
+        mobile_atoms = mobile.select_atoms(subselection)
     else:
         try:
-            atoms = subselection.atoms
+            mobile_atoms = subselection.atoms
         except AttributeError:
             raise TypeError("subselection must be a selection string, a AtomGroup or Universe or None")
 
-    atoms.translate(-mobile_com)
-    atoms.rotate(R)
-    atoms.translate(ref_com)
+    #_fit_to DOES subtract center of mass, will provide proper min_rmsd
+    mobile_atoms, new_rmsd = _fit_to(mobile_coordinates,\
+        ref_coordinates, mobile_atoms, mobile_com, ref_com, weights=weights)
 
+    logger.info('old_rmsd: {0} new_rmsd: {1}'.format(old_rmsd,new_rmsd))
```

Comment:
nah to much logging you are returning those values. The log should contain important messages for the user when something awesome happened or things went wrong.